ik_solver_msgs/IkTarget[] targets

string frame_id
string[] seed_joint_names
uint32 max_number_of_solutions # if zero, use default.
uint32 stall_iterations # if zero, use default.

bool parallelize
bool exploit_solutions_as_seed
---
ik_solver_msgs/IkSolution[] solutions
string[] joint_names
